contracts/src/CrossChainApplications/ERC20Bridge/README.md

```diff
@@ -13,7 +13,7 @@ The generic ERC20 bridge is implemented using two primary contracts.
     - Bridge contract that uses Teleporter to facilitate the bridging operations of adding new supported tokens and moving tokens between chains.
     - The bridge contract tracks the balances of each token sent to other bridge instances, and only allows those bridge instances to redeem up to the amount of tokens that has been previously sent to them.
     - Primary functions include:
-        - `submitCreateBridgeToken`: Called on the origin subnet to add support for an ERC20 native to the chain to a bridge instance on a different chain. Submits a Teleporter message that invokes `createBridgeToken` on the destination chain.
+        - `submitCreateBridgeToken`: Called on the origin chain to add support for an ERC20 native to the chain to a bridge instance on a different chain. Submits a Teleporter message that invokes `createBridgeToken` on the destination chain.
```

Comment:
```suggestion
        - `submitCreateBridgeToken`: Called on the origin chain to add support for a native ERC20 token on a different chain's bridge instance. Submits a Teleporter message that invokes `createBridgeToken` on the destination chain.
```

Reply:
"Native" hear can be a bit confusing, since it's possible for the ERC20 on a source chain to not be "native" to that chain if it was bridged to that chain by a bridging protocol. For example, USDC.e on Avalanche is not "native" to the C-Chain, even though it's very likely that we'd bridge USDC.e from the C-Chain to a subnet via a bridge like this.

Reply:
Will remove native altogether, and reword the other part.